charts/gateway/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
-appVersion: "3.2.2"
+appVersion: "3.3.0"
 description: Conduktor Gateway chart
 name: conduktor-gateway
-version: 3.2.2
+version: 3.3.0
```

Comment:
Don't update chart version on PR, it will be done by the [release workflow](https://github.com/conduktor/conduktor-public-charts/actions/workflows/create-release.yaml)